global.json

```diff
@@ -0,0 +1,5 @@
+{
+  "sdk": {
+    "version": "3.0.100"
+  }
+}
```

Comment:
What is the reason for adding global.json? It forces using a certain version of SDK instead of using latest. I see no reason to force us to use just this specific version of .NET Core SDK.

Reply:
discussed: For this application we prefer not to use common team settings via global.json.